x/ibc/light-clients/07-tendermint/types/misbehaviour_handle.go

```diff
@@ -81,13 +77,13 @@ func (cs ClientState) CheckMisbehaviourAndUpdateState(
 	// be nil since the previous param values will be used in case they can't be
 	// retrieved. If they are not set during initialization, Tendermint will always
 	// use the default values.
-	if consensusParams != nil &&
-		consensusParams.Evidence != nil &&
-		(ageDuration > consensusParams.Evidence.MaxAgeDuration ||
-			ageBlocks > consensusParams.Evidence.MaxAgeNumBlocks) {
+	if cs.ConsensusParams != nil &&
```

Comment:
I'm not sure if this satisfies the note above: `"If they are not set during initialization, Tendermint will always use the default values."`.

I assume from that that if the `cs.ConsensusParams` you would compare the values against a default set from tendermint.

Reply:
Do we know for certain the comment is correct?

@AdityaSripal @cwgoes thoughts on how to handle chains that don't set a consensus state at the application level?

Reply:
We probably shouldn't allow `nil` consensus parameters at all? Seems like we can't validate misbehaviour otherwise